packages/react/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,109 @@
+import React, {
+  useState,
+  useRef,
+  useEffect,
+  KeyboardEventHandler
+} from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import Scrim from '../Scrim';
+import Icon from '../Icon';
+import { useId } from 'react-id-generator';
+
+interface NavBarProps {
+  children: React.ReactNode;
+  initialActiveIndex?: number;
+  className?: string;
+  collapsed?: boolean;
+  navTriggerLabel?: string;
+  propId?: string;
+}
+
+const NavBar = ({
+  children,
+  className,
+  collapsed = false,
+  navTriggerLabel = 'MAIN MENU',
+  propId
+}: NavBarProps) => {
+  const navRef = useRef<HTMLElement>(null);
+  const triggerRef = useRef<HTMLButtonElement>(null);
+  const [showDropdown, setShowDropdown] = useState(false);
+  const [menuId] = [propId] || useId(1, 'navbar');
+  const showNavItems = !collapsed || (collapsed && showDropdown);
+
+  const handleOutSideEvent = (e: FocusEvent | MouseEvent) => {
+    const target = e.target as HTMLElement;
+    if (navRef.current && !navRef.current?.contains(target)) {
+      setShowDropdown(false);
+    }
+  };
+
+  const handleKeyDown: KeyboardEventHandler<HTMLUListElement> = (
+    e: React.KeyboardEvent<HTMLUListElement>
+  ) => {
+    if (e.key !== 'Escape') {
+      return;
+    }
+
+    setShowDropdown(false);
+    triggerRef.current?.focus();
+  };
+
+  useEffect(() => {
+    if (collapsed && showDropdown) {
+      document.addEventListener('focusin', handleOutSideEvent);
+
+      return () => {
+        document.removeEventListener('focusin', handleOutSideEvent);
+      };
+    }
+  }, [collapsed, showDropdown]);
+
+  const handleTriggerClick = () => {
+    setShowDropdown(!showDropdown);
+  };
+
+  return (
+    <nav
+      className={classNames('NavBar', className, {
+        'NavBar--collapsed': collapsed
+      })}
+      ref={navRef}
+    >
+      <Scrim show={showDropdown} />
```

Comment:
I don't see the scrim tied to this component anywhere in the uxpins....I think we should remove it because with a scrim comes a whole bunch of a11y considerations (like preventing screen readers from traversing to content behind scrim)


```suggestion
```

Reply:
just confirmed with the designers and we should in fact remove the scrim here